File: 은지/LV2/최댓값과 최솟값.js
function solution(s) {
  var numbers = [];
  var sub = ""
  for (let v of s) {
    if (v !== " ") {
      sub += v
    } else {
      numbers.push(Number(sub))
      sub = ""
    }
  }
  numbers.push(Number(sub))
  var answer = String(Math.min(...numbers)) + " " + String(Math.max(...numbers))
  return answer;
}

File: 은지/LV2/최댓값과 최솟값.py
def solution(s):
    numbers = []
    sub = ""
    for v in s:
        if v != " ":
            sub += v
        else:
            numbers.append(int(sub))
            sub = ""
    numbers.append(int(sub))
    answer = str(min(numbers)) + " " + str(max(numbers))
    return answer

s = "1 2 3 4"
print(solution(s))